composables/auto-prompter.ts

```diff
@@ -294,25 +301,193 @@ export function useAutoPrompter() {
 
         newBattle.winner = winner
         battles.value.push(newBattle)
+    }
 
-        takeSnapshotOfRatings()
+    // define ammount of battles and prompts in each round
+    function defineBattlesPerRound (numberOfPrompts: number, numberOfBattles: number) {
+        const promptsPerRound: number[] = []
+        const battlesPerRound: number[] = []
+        let sumOfPromptsPerRound = 0
+        let sumOfBattlesPerRound = 0
+
+        // calculate number of prompts that will be alive in each round
+        promptsPerRound[0] = numberOfPrompts
+        for (let i = 1; i < Math.floor(Math.log2(numberOfPrompts)); i++) {
```

Comment:
```suggestion
	// The amount of brackets, where a bracket is a set of battles with a unique amount of prompt candidates
	// So if the initial amount of prompt candidates is 8, the bracket amount will be 3
	// Because:
	// 	From 8 prompts, 4 will remain
	//	From 4 prompts, 2 will remain
	// 	From 2 prompts, 1 will remain
	const bracketAmount = Math.floor(Math.log2(numberOfPrompts))
        for (let i = 1; i < bracketAmount; i++) {
```